### PR TITLE
Issue/2559 Display all indicators/observables contained in reports which contain a specific entity(#2559)

### PR DESCRIPTION
**Pull request by marieflorescontact**
This is the 1st part of the issue applied to the `Threat Actor Individual`, once approved we will extend it to the rest of the entities concerned.

### Related Brainstorm

* [Brainstorm](https://www.notion.so/filigran/Display-all-indicators-observables-contained-in-reports-which-contain-a-specific-entity-5f17d4fc1fd04cfbbf87594288beeec8)

### Explanation of code structure and opening on the next step

* [Explanation](https://www.notion.so/filigran/Display-all-indicators-observables-contained-in-reports-which-contain-a-specific-entity-5f17d4fc1fd04cfbbf87594288beeec8?pvs=4#62d37188187f46d49d5d390c00b441ae)

### Proposed changes

* Remove `Indicators` tab from the knowledge Topbar
* `Indicators` are now accesible from the knowledge right bar above `Observables`
*  Add `contextual view` that displays `Observables/Indicators` that are contained in `Reports/Case/Grouping` that contains the Entity
* Create reusable `view component ` to display `EntistixCorerelationships` : `EntitiesView` / `RelationshipView` / `ContextualView`
* Add filtering on `Pattern Type` and `Indicator type` for Indicators entities view

### Related issues

* [Display all indicators/observables contained in reports which contain a specific entity](https://github.com/OpenCTI-Platform/opencti/issues/2559)

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yassine-ouaamou — COMMENTED** [review]
When adding a relationship with indicators, if we do it from "Entities view", we have the possiblity to create "indicates" or "related to" relationships.
If done from "Relationships views", only "indicates" is possible.
possible relationships

Is it normal?

**Comment by yassine-ouaamou** [comment]
- The columns are misaligned in the "Entites view"

image

- In the "Relationship view", the links to the indicators are broken.
- "Related entities" seems to be broken too. Maybe it's just me

File: opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.graphql [hunk 1]
```diff
@@ -85,7 +85,7 @@ type ThreatActorIndividual implements BasicObject & StixObject & StixCoreObject
     filters: [StixCoreRelationshipsFiltering]
     filterMode: FilterMode
   ): [Distribution] @auth(for: [KNOWLEDGE, EXPLORE])
-  # StixDomainObject
+  stixCoreObjectFromContainer(fromId: String!, entityType: String!, containerType: String!): StixCoreObjectConnection @auth(for: [KNOWLEDGE])# StixDomainObject
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.graphql]: Doesn't seem to be used anywhere

Comment (marieflorescontact) [reply opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.graphql]: fixed

**Comment by marieflorescontact** [comment]
**PR status**

- all previous comments have been taken into account.

**1 new issue to be fixed**:
- It crashes when sorting by `relationship_type` on the relationship view and switching to the contextual view

**Comment by marieflorescontact** [comment]
**PR status**

- all previous comments have been taken into account.
- bug on ordering fixed

PR ready for review

**Comment by Archidoit** [comment]
Overview of a threat actor individual: bad display of 'details' (content of 'goals' in the second column whereas the 'goals' title is in the first one, bad seperation after the '-' of description...) :
image

**Comment by Archidoit** [comment]
- in a threat actor individual > Knowledge > Indicator > contextual view : bad alignement of the selection squares:
image

- missing translation: 'Relationships view'
image

File: opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.graphql [hunk 2]
```diff
@@ -85,7 +85,6 @@ type ThreatActorIndividual implements BasicObject & StixObject & StixCoreObject
     filters: [StixCoreRelationshipsFiltering]
     filterMode: FilterMode
   ): [Distribution] @auth(for: [KNOWLEDGE, EXPLORE])
-  # StixDomainObject
```
Comment (yassine-ouaamou) [inline opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.graphql]: Can you put back this comment, please?

**Comment by RomuDeuxfois** [comment]
**PR status**

- all previous comments have been taken into account.

PR ready for review 🔥 🔥 🔥

**Review by yassine-ouaamou — APPROVED** [review]

**Review by Archidoit — APPROVED** [review]

**Pull request merged**